front/.storybook/main.ts

```diff
@@ -0,0 +1,26 @@
+import type { StorybookConfig } from "@storybook/nextjs";
```

Comment:
should the `.storybook` folder be included in the commited code ?
I guess so but just checking in case

Reply:
Yep, it contains the base component used to preview the content, allowing personalization if needed